docs/usecase/usage-example.md

```diff
@@ -0,0 +1,79 @@
+# Use cases
+
+  This document will introduce you to the example of what Vald can do.
+  Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine, which uses [NGT](https://github.com/yahoojapan/NGT) as the core engine of Vald, and Vald manages to integrate with Kubernetes.
+
+  In general, you cannot search your unstructured data using the inverted index, like images and videos.
+  By applying the model like BERT or VGG, you can convert your unstructured data into vectors.
+  After converting them into vectors, you can insert it to Vald and process them in Vald.
+
+  Here are some general use cases of Vald or vector search engine.
+
+## Image and video processing
+
+  You can use Vald as the image/video processing engine to search the similar image/video or analysis the image/video for your use case.
+
+  Vald is capable to process a huge number of images at the same time so it case fit with your use case.
+
+  Here are some examples of what you can do with images and videos in Vald.
```

Comment:
I think it is correct to say "using Vald" because these example cannot achieve by using only Vald.
```suggestion
  Here are some examples of what you can do with images and videos using Vald.
```